packages/amf/amf.0.1.2/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+synopsis: "Parser/serializer for Adobe's Action Message Format"
+maintainer: "Brian Caine <brian.d.caine@gmail.com>"
+authors: "Brian Caine <brian.d.caine@gmail.com>"
+license: "LGPL with OCaml linking exception"
+homepage: "https://briancaine.github.io/ocaml-amf/"
+bug-reports: "https://github.com/briancaine/ocaml-amf/issues"
+depends: [
+  "ocaml" {>= "4.01"}
```

Comment:
```suggestion
  "ocaml" {>= "4.03"}
```
Your package fails on OCaml 4.03:
```
#=== ERROR while compiling amf.0.1.2 ==========================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.03.0 | pinned(https://github.com/briancaine/ocaml-amf/archive/v0.1.2.tar.gz)
# path                 ~/.opam/4.03/.opam-switch/build/amf.0.1.2
# command              ~/.opam/4.03/bin/dune build -p amf -j 2
# exit-code            1
# env-file             ~/.opam/log/amf-15-fd06be.env
# output-file          ~/.opam/log/amf-15-fd06be.out
### output ###
#       ocamlc src/amf/.AMF.objs/byte/aMF__Amf_v0.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.03/bin/ocamlc.opt -w -40 -g -bin-annot -I src/amf/.AMF.objs/byte -I /home/opam/.opam/4.03/lib/base -I /home/opam/.opam/4.03/lib/base/caml -I /home/opam/.opam/4.03/lib/base/shadow_stdlib -I /home/opam/.opam/4.03/lib/bin_prot -I /home/opam/.opam/4.03/lib/bin_prot/shape -I /home/opam/.opam/4.03/lib/bisect_ppx/runtime -I /home/opam/.opam/4.03/lib/bytes -I /home/opam/.opam/4.03/lib/core -I /home/opam/.opam/4.03/lib/core_kernel -I /home/opam/.opam/4.03/lib/core_kernel/base_for_tests -I /home/opam/.opam/4.03/lib/fieldslib -I /home/opam/.opam/4.03/lib/jane-street-headers -I /home/opam/.opam/4.03/lib/num -I /home/opam/.opam/4.03/lib/ocaml/threads -I /home/opam/.opam/4.03/lib/ppx_assert/runtime-lib -I /home/opam/.opam/4.03/lib/ppx_bench/runtime-lib -I /home/opam/.opam/4.03/lib/ppx_compare/runtime-lib -I /home/opam/.opam/4.03/lib/ppx_expect/collector -I /home/opam/.opam/4.03/lib/ppx_expect/common -I /home/opam/.opam/4.03/lib/ppx_expect/config -I /home/opam/.opam/4.03/lib/ppx_hash/runtime-lib -I /home/opam/.opam/4.03/lib/ppx_inline_test/config -I /home/opam/.opam/4.03/lib/ppx_inline_test/runtime-lib -I /home/opam/.opam/4.03/lib/sexplib -I /home/opam/.opam/4.03/lib/sexplib/0 -I /home/opam/.opam/4.03/lib/sexplib/unix -I /home/opam/.opam/4.03/lib/spawn -I /home/opam/.opam/4.03/lib/stdint -I /home/opam/.opam/4.03/lib/stdio -I /home/opam/.opam/4.03/lib/typerep -I /home/opam/.opam/4.03/lib/variantslib -no-alias-deps -open AMF__ -o src/amf/.AMF.objs/byte/aMF__Amf_v0.cmo -c -impl src/amf/amf_v0.pp.ml)
# File "src/amf/amf_v0.ml", line 83, characters 40-55:
# Error: Unbound value Bytes.of_string
```

Reply:
https://travis-ci.org/ocaml/opam-repository/jobs/582908720#L403

So I changed it to `"ocaml" {> "4.03"}`, to exclude 4.03, but it's still running the tests for 4.03. How can I exclude that?

Reply:
Sorry my bad, I meant to say >= 4.04